inlong-manager/manager-service/src/main/java/org/apache/inlong/manager/service/user/TenantRoleServiceImpl.java

```diff
@@ -137,4 +139,14 @@ public Boolean delete(Integer id) {
         return true;
     }
 
+    @Override
+    public int saveDefault(String username, String operator) {
```

Comment:
Please change the return type to `Integer` in service or controller layer, thanks.

Reply:
fixed, thx